mlocal/frags/Makefile.stub

```diff
@@ -194,7 +194,9 @@ collect:
 check:
 	@echo " CHECK go fmt"
 	$(V)export CGO_CPPFLAGS="$(cgo_CPPFLAGS)" CGO_LDFLAGS="$(cgo_LDFLAGS)" && \
-		cd $(SOURCEDIR) && test -z $(go fmt ./...)
+		cd $(SOURCEDIR) && \
+		( test -z "`go fmt -n ./... | sed 's/ -w / /' | sh | tee /dev/stderr`" || \
+		   ( echo "Use 'go fmt' to update formatting"; false ) )
```

Comment:
```
( echo "The following files have formatting errors:" ) && \
( echo gofmt -l `go list -f "{{.Dir}}" ./...` ) && \
( echo "Use 'go fmt github.com/singularityware/singularity/...' to correct all formatting errors"; false ) )
```

Reply:
I updated the message to something similar.